src/rules/selector-no-empty/README.md

```diff
@@ -0,0 +1,47 @@
+# selector-no-empty
+
+Disallow empty selectors.
+
+```css
+    a { color: pink; }
+/** ↑ 
+ * This is selector */
+```
+
+Empty selectors (by themselves or within a selector list) invalidate a rule. This rule splits the selector list on, and any part of zero length or contains only whitespace is consider empty. Everything else isn't.
```

Comment:
"Empty selectors (by themselves or within a selector list) invalidate a rule. This rule splits selector lists on `,`. Any part of zero length or containing only whitespace is deemed empty."